src/Admin/Models/ProviderEditModel.cs

```diff
@@ -17,14 +19,22 @@ public ProviderEditModel(Provider provider, IEnumerable<ProviderUserUserDetails>
             Enabled = provider.Enabled;
         }
 
-        public string Administrators { get; set; }
-
         public bool Enabled { get; set; }
 
         public string BillingEmail { get; set; }
 
         public string BusinessName { get; set; }
 
         public string Name { get; set; }
+        
+        public Provider ToProvider(Provider existingProvider)
+        {
+            existingProvider.Name = Name;
+            existingProvider.BusinessName = BusinessName;
+            existingProvider.BillingEmail = BillingEmail?.ToLowerInvariant()?.Trim();
+            //existingProvider.UseEvents = UseEvents;
```

Comment:
Is this commented out line needed?

Reply:
Went back and added the checkbox I was to lazy to do when I commented that code.